arelle/plugin/inlineXbrlDocumentSet.py

```diff
@@ -86,14 +89,14 @@ def inlineXbrlDocumentSetLoader(modelXbrl, normalizedUri, filepath, isEntry=Fals
     if isEntry:
         try:
             if "entrypoint" in kwargs:
-                _target = kwargs["entrypoint"]["ixdsTarget"]
+                _target = kwargs["entrypoint"].get("ixdsTarget") # assume None if not specified in entrypoint
             elif "ixdsTarget" in kwargs: # passed from validate (multio test cases)
                 _target = kwargs["ixdsTarget"]
             else:
-                _target = modelXbrl.modelManager.formulaOptions.parameterValues.get("ixdsTarget")[1]
+                _target = modelXbrl.modelManager.formulaOptions.parameterValues["ixdsTarget"][1]
```

Comment:
Should this be checked for existence? I am also unclear why specifying `entrypoint` is mutually exclusive with specifying `ixdsTarget` if the `entrypoint.ixdsTarget` isn't set. I would expect something like this:

```python
_target = None

if 'ixdsTarget' in kwargs.get('entrypoint', {}):
    ...
elif 'ixdsTarget' in kwargs:
   ...
elif len(modelXbrl.modelManager.formulaOptions.parameterValues.get('ixdsTarget', [])`
   ...
modelXbrl.ixdsTarget = None if _target == '(default)' else _target or None
```

to avoid NPE's and provide fallbacks

Reply:
Originally if entrypoint is used, ixdsTarget was only specified for non-default targets and the default was the absence of ixdsTarget.  However if we want so respecify that ixdsTarget has to be "(default)" to recognize its use in entrypoint then the above structure makes more sense and the ixdsTarget in parameters can be used when no ixdsTarget in entrypoint.  Since this is probably not yet used by anybody except validation conformance suite (where it comes in kwargs), maybe we could or should change.  Discuss on stand up?  Code not changed today.